python/llm/example/CPU/StableDiffusion/README.md

```diff
@@ -0,0 +1,40 @@
+# Stable Diffusion
+In this directory, you will find examples on how to run StableDiffusion models on CPU.
+
+### 1. Installation
+#### 1.1 Installation on Linux
```

Comment:
#### 1.1. Install IPEX-LLM 
Follow the instructions in [IPEX-LLM CPU installation guide](https://ipex-llm.readthedocs.io/en/latest/doc/LLM/Overview/install_cpu.html) to install ipex-llm. We recommend to use miniconda to manage your python environment. 

#### 1.2 Install dependencies for Stable Diffusion
Assume you have created a conda environment named `diffusion` with `ipex-llm` installed. Run below commands to install dependencies for running Stable Diffusion.  
```python
conda activate `diffusion`
pip install ...
...
```